### DISCUSSION TITLE
Web Preview Error Message

### DISCUSSION THREAD

**Discussion by annamorales5** [thegetty/quire#221] [Get Help] [answered]
Hello! I am just getting started with using Quire to build a site, and am getting this error message on my Terminal when I try to open my new project:

"Error: Error building site: failed to load data: failed to load data: "/Users/annamorales/my-project/data/publication.yml:18:1": failed to unmarshal YAML: yaml: line 18: did not find expected key
Run `quire preview --verbose` for details"

If anyone has any idea how to fix this, I would greatly appreciate any help! Thank you.

**Comment by geealbers** [comment]
Hi @annamorales5, the wording on that error is a little cryptic, but what it's trying to tell you is that there is a yaml validation error on line 18 of your publication.yml file (that's the "data/publication.yml:18:1" bit). Most often this is a colon that is somewhere it shouldn't be, or an indentation that's a little off. There are some [tips in the documentation](https://quire.getty.edu/documentation/fundamentals/#yaml-basics) that might help track the error, and an [online YAML validator](https://codebeautify.org/yaml-validator) can help too. Let us know if this helps!

**Reply by annamorales5** [reply]
Hey Greg! @geealbers Thanks for this- this is helpful. I think this issue might've been created when I copied and pasted some text into one of the lines last time we spoke, and ended up deleting it because I believe I entered it incorrectly. It is possible that I accidentally deleted something that needs to be there for the .yaml to be properly coded when doing this? Now, terminal is saying that it "did not find an expected key" in line 17, which is currently empty in Atom.

**Reply by geealbers** [reply]
@annamorales5, could you share the top 20 lines or so of your publication.yml file here?

**Reply by annamorales5** [reply]
<img width="967" alt="Screen Shot 2021-12-07 at 3 51 11 PM" src="https://user-images.githubusercontent.com/93948401/145104211-22c6132c-3420-45ec-ab32-7cf07729f578.png">

Yes- here is a screenshot @geealbers.

**Reply by geealbers** [reply]
Try deleting live 17 altogether. The structure of the data here is meant to indicate a `one_line` description and a `full` description. Having a line break between those two terms and the `description` term on line 16, is sort of disconnecting them. In the end, it should be structured like lines 31–32, where you have a main term followed immediately by two sub-terms.

**Comment by annamorales5** [comment]
<img width="871" alt="Screen Shot 2021-12-07 at 7 32 59 PM" src="https://user-images.githubusercontent.com/93948401/145127123-550fa11f-f16c-4c77-93f6-a1c917db5b87.png">
I see @geealbers thank you. I deleted line 17 so it looks more like lines 31-31. Now I am getting the same error message for line 16 (it seems to be making its way up the line hahaha). I pasted my text in the yaml validator and it was valid, and I read through the documentation tips you suggested but cannot figure out why there is an issue with this line as well. Any ideas?

**Reply by geealbers** [reply]
That's weird! I don't see the issue either. Can you attach the complete publication.yml file here so I can try it in another project, and also send along the exact Terminal output you're currently getting?

**Comment by Erin-Cecele** [comment]
Hi @annamorales5 I think I see the issue. The "description" should be blank. That line serves as more of a heading, whereas the fields that fall beneath it are where the content should go. 

Here is an example of the proper formatting:

<img width="763" alt="Screen Shot 2021-12-07 at 4 47 11 PM" src="https://user-images.githubusercontent.com/62075223/145128474-9c5047e8-9e02-4f09-87d2-f345bd511bd3.png">

**Comment by annamorales5** [comment]
Erin & Greg,

This worked!!! Thanks very much for your advice and patience as I slowly but surely learn how to navigate this. I am looking forward to moving with my project. Hope you are feeling better, Erin 🙂.

Best,
Anna Morales

-------------------------------------------------------------------------------------------------------------------------------------------------
Anna Morales
Photographer & Visual Anthropologist
Cell: 267-901-7001
Columbia, SC & Philadelphia, PA
https://avisualanthropologist.com/





________________________________

**Comment by mmpadilla15** [comment]
@geealbers and @Erin-Cecele - I was having the same problem and this helped me to pinpoint it. I think I've got an image caption that maybe has a few too many colons and quotes incorporated into it. When I take it out, preview runs fine, but when I add it back in, I get the error. This is my caption in line 186:
<img width="1385" alt="Screen Shot 2022-08-23 at 2 00 05 PM" src="https://user-images.githubusercontent.com/74433007/186246190-e3d2e116-c97f-429d-ae12-b678691d3097.png">

Can you advise on how I should format the caption so that the preview renders properly?

**Comment by Erin-Cecele** [comment, marked as answer]
Hi @mmpadilla15. Your erorr is being caused by the use of straight quotes inside the caption: "Counterpoise" and "Ode to a Cottonpicket". This can be remedied but adding curly quotes. I can't remember if you are on a Mac or PC but if you are on a Mac you can highlight the text you want to wrap in quotes, and press Option and left bracket [ at the same time. If you are on a PC, check out this helpful article: [https://www.lifewire.com/typing-quotes-apostrophes-and-primes-1074104](https://www.lifewire.com/typing-quotes-apostrophes-and-primes-1074104). Let us know if that fixes your issue.

**Reply by mmpadilla15** [reply]
OMG ... I knew it had to be something super-simple like that! It's working now - thank you so much!

**Reply by Erin-Cecele** [reply]
Excellent!! Great news, Michelle!
